.github/workflows/translation.yml

```diff
@@ -0,0 +1,81 @@
+name: "Refresh translations"
+
+on:
+  workflow_call:
+    secrets:
+      tx_token:
+        required: true
+      based_branch:
+        required: true
+
+jobs:
+  ci:
+    name: "Translations"
+    runs-on: ubuntu-latest
+    steps:
+
+    ##########################################
+    # Checkout repository                    #
+    ##########################################
+    - name: "Checkout"
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ secrets.based_branch }}
+        fetch-depth: 0 #To Fetch All Tags and Branches
+
+    ##########################################
+    # Install depdendencies                  #
+    ##########################################
+    - name: Install Dependencies
+      uses: php-actions/composer@v6
+      with:
+        dev: yes
+
+    - name: Install xgettext
+      run: sudo apt-get install gettext libgettextpo-dev
+
+    ##########################################
+    # Generate <plugin>.pot file             #
+    ##########################################
+    - name: "Generate <plugin>.pot file"
+      run: ./vendor/bin/extract-locales
+
+
+    ##########################################
+    # Push and Pull transifiex source        #
+    ##########################################
+    - name: Push source file (.pot)
+      uses: transifex/cli-action@v2
+      with:
+        args: push
+        token: ${{ secrets.tx_token }}
+
+    - name: Clean Transifex CLI
+      run: rm -fr /tmp/tx
```

Comment:
Because  of the implementation of the githuab action ....

eEch time it is called, it create temporary directory to store the transifex CLI

https://github.com/transifex/cli-action/blob/master/action.yaml#L18-L30

This results in the following error

`mkdir: cannot create directory ‘tx’: File exists`